src/Umbraco.Core/Configuration/Models/ImagingCacheSettings.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.IO;
+
+namespace Umbraco.Core.Configuration.Models
+{
+    public class ImagingCacheSettings
+    {
+        public int MaxBrowserCacheDays { get; set; } = 7;
+
+        public int MaxCacheDays { get; set; } = 365;
+
+        public uint CachedNameLength { get; set; } = 8;
+
+        public string CacheFolder { get; set; } = Path.Combine("..", "Umbraco", "MediaCache");
```

Comment:
Same question re tilda on paths ~

Reply:
The config is passed directly to ImageSharp config, We currently dont have a good way to interrupt and parse the "~". If we use "~" it creates a folder named "~" in wwwroot